### PR TITLE
Added spanish translation for README.md

### LINKED ISSUE #1943: Seeking SPANISH translation for Readme.md
Use our [translation guide](https://yahoo.github.io/hacktogether/html/contribute.html#translation) to provide a translation in SPANISH  for the following files: 

[README.md ](https://github.com/yahoo/elide/blob/master/README.md)

### PR DESCRIPTION
**Pull request by JFernando122**
Resolves #1943

## Description
Added spanish translation for `README.md` file

## Motivation and Context
Completly translated the main `README.md` file into the file at `/translations/es/README.md` except for the names of the articles at the very bottom. I thought it would be better to keep those in english but can translate them too if need it.

Also i noticed that on the other translation they translated the `CONTRIBUTING.md` file as well, should i do the same?

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: translations/es/README.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
+# Elide
+
+> _API's obstinados para aplicaciones móviles y web._
```
Comment (aklish) [inline translations/es/README.md:3]: What is the difference between obstinados and dogmáticos in Spanish?  Google translate says this is 'stubborn', but I'm no Spanish expert.

File: translations/es/README.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+## Instalación
+
+Para probar un ejemplo de servicio de Elide, revise este [Projecto de ejemplo en Spring Boot](https://github.com/yahoo/elide-spring-boot-example).
+
+Alternativamente use [elide-standalone](https://github.com/yahoo/elide/tree/master/elide-standalone) que permite una configuración rápida de una instancia local de Elide ejecutándose de manera embebida en una aplicación Jetty.
+
+## Uso
+
+Los siguientes ejemplos aprovechan elide 5 (ahora en pre-lanzamiento). Para documentación en versiones anteriores/estables, visite [aquí](https://elide.io/pages/guide/v4/01-start.html).
```
Comment (aklish) [inline translations/es/README.md:79]: I forgot to remove this blurb after Elide 5 release.  We can remove the following text altogether:

"Los siguientes ejemplos aprovechan elide 5 (ahora en pre-lanzamiento). Para documentación en versiones anteriores/estables, visite [aquí](https://elide.io/pages/guide/v4/01-start.html)."

**Comment by aklish** [comment]
I'm working to fix the Elide 5 build so we can merge the PR.  Translation looks great.  I had a few questions and noted one section of stale text that can be removed.  

If you have cycles, would love to also have the CONTRIBUTING.md translated as well.  

Thanks again!

File: translations/es/README.md (hunk 1 above)
Comment (JFernando122) [reply translations/es/README.md:3]: they are basically the same.
I think the main difference is that dogmáticos refers to both religion or technological **systems** and obstinados is more general, should i switch one of them to keep it consistent?

File: translations/es/README.md (hunk 2 above)
Comment (JFernando122) [reply translations/es/README.md:79]: ok will remove it and start working on the CONTRIBUTING.md file as well

**Comment by aklish** [comment]
I'm going to merge this PR as is.  If you have cycles to add CONTRIBUTING.md, that can be done in a separate PR.  Thanks again for this.

**Review by aklish — APPROVED** [review]

**Pull request merged**